user/hare-update/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "hare-update"
+pkgver = "0.25.2.0"
+pkgrel = 0
+build_style = "makefile"
+make_install_args = ["LIBEXECDIR=/usr/lib"]
```

Comment:
```suggestion
make_install_args = ["LIBEXECDIR=/usr/lib"] # XXX libexecdir
```
ditto for the other templates

Reply:
just a small convention, it's done pretty much everywhere else too

Reply:
i don't see the point but i added it anyway